mixed/spark/v3.3/spark-runtime/pom.xml

```diff
@@ -63,7 +63,7 @@
                                     <include>com.netease.amoro:amoro-core</include>
                                     <include>com.netease.amoro:amoro-ams-api</include>
                                     <include>com.netease.amoro:amoro-mixed-hive</include>
-                                    <include>com.alibaba:fastjson</include>
+                                    <include>com.fasterxml.jackson.core:jackson-databind</include>
```

Comment:
Why does Spark 3.3 require relocation of this class, while Spark 3.2 does not?

Reply:
maybe after shade the jackson we involed, we can remove this here.

IIUC, this added because of failed test